fearless/Modules/Staking/SelectValidatorsFlow/ValidatorSearch/ValidatorSearchInteractor.swift

```diff
@@ -1,7 +1,43 @@
-final class ValidatorSearchInteractor {}
+import Foundation
+import RobinHood
+
+final class ValidatorSearchInteractor {
+    weak var presenter: ValidatorSearchInteractorOutputProtocol!
+
+    let validatorOperationFactory: ValidatorOperationFactoryProtocol
+    let operationManager: OperationManagerProtocol
+
+    init(
+        validatorOperationFactory: ValidatorOperationFactoryProtocol,
+        operationManager: OperationManagerProtocol
+    ) {
+        self.validatorOperationFactory = validatorOperationFactory
+        self.operationManager = operationManager
+    }
+}
 
 extension ValidatorSearchInteractor: ValidatorSearchInteractorInputProtocol {
-    func setup() {
-        #warning("Not implemented")
+    func performValidatorSearch(accountId: AccountId) {
+        let operation = validatorOperationFactory
```

Comment:
What if we already searching in that case race condition can happen. I think we should store pending wrapper locally and cancel it in case new one comes.